app/Auth0/CachedAuth0ClientGrants.php

```diff
@@ -16,6 +18,7 @@ public function findGrantsOnClient(Auth0Client $auth0Client): array
     {
         if(! isset($this->grants[$auth0Client->clientId])) {
             $this->grants[$auth0Client->clientId] = $this->sdk->findGrantsOnClient($auth0Client);
+            Log::debug('CachedAuth0ClientGrants  - ' . $auth0Client->clientId . ': cache miss');
```

Comment:
I think a cache hit is more important because a cache miss is already logged through `findGrantsOnClient`